src/components/customerCases/customerCase/sections/results/resultsBlock.module.css

```diff
@@ -93,6 +94,28 @@
   border-top-left-radius: 0;
 }
 
-.highlightRow .highlightCard {
-  width: 100%;
+@media (max-width: 640px) {
```

Comment:
Any reason for breaking at 640px? We have defined breaking points at 834 for tablet and 425 for mobile, so just wondering why 640px was used here :)

Reply:
just a derp, saw 640 being used somewhere 
<img width="265" alt="image" src="https://github.com/user-attachments/assets/07d76754-12f1-41da-ac40-92f74d5aa637">